justfile

```diff
@@ -0,0 +1,38 @@
+set dotenv-load
+set positional-arguments
+
+
+test_dsl:
+  #!/usr/bin/env bash
+  set -euxo pipefail
+
+  PGPASSWORD=$CS_DATABASE__PASSWORD dropdb --force --if-exists --username $CS_DATABASE__USERNAME --port $CS_DATABASE__PORT cs_migrator_test
+  PGPASSWORD=$CS_DATABASE__PASSWORD createdb --username $CS_DATABASE__USERNAME --port $CS_DATABASE__PORT cs_migrator_test
+
+  connection_url=postgresql://$CS_DATABASE__USERNAME:@localhost:$CS_DATABASE__PORT/cs_migrator_test
+  PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f sql/dsl-core.sql
+  PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f sql/dsl-config-schema.sql
+  PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f sql/dsl-config-functions.sql
+  PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f sql/dsl-encryptindex.sql
+
+  # tests
+  # PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f tests/core.sql
+  PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f tests/config.sql
+  # PGPASSWORD=$CS_DATABASE__PASSWORD psql $connection_url -f tests/encryptindex.sql
```

Comment:
Why are the core and encryptindex tests commented out? Is that intentional?

Reply:
oops, no, we just commnted them out to reduce noise. I'll revert